bg_mon.c

```diff
@@ -530,9 +530,9 @@ static struct evbuffer *prepare_statistics_output(struct timeval time, system_st
 #endif
 			if (s.state == STATE_IDLEINTRANSACTION && s.idle_in_transaction_age > 0) {
 				if (s.idle_in_transaction_age < 10)
-					evbuffer_add_printf(evb, ",\"query\":\"idle in transaction %.2g\"", s.idle_in_transaction_age);
+					evbuffer_add_printf(evb, ",\"query\":%s ,\"idle in transaction\" %.2g", s.query, s.idle_in_transaction_age);
 				else
-					evbuffer_add_printf(evb, ",\"query\":\"idle in transaction %ld\"", (long)s.idle_in_transaction_age);
+					evbuffer_add_printf(evb, ",\"query\":%s ,\"idle in transaction\" %.ld", s.query, (long)s.idle_in_transaction_age);
```

Comment:
@RafiaSabih please extract the `query` print to a separate line. I.e. before https://github.com/CyberDem0n/bg_mon/pull/46/files#diff-57f1b76e196394f418a39a1f6d499e8da40bc02a013bf7db13a092fa79c1a60aR532